packages/styles/src/core/layout/sidebar/README.md

```diff
@@ -1,103 +1,137 @@
-## Sidebar
+## .as-sidebar
 
-Sidebars are the area where your app content lives. Sidebars are located beside the map wrapper and should be placed inside an element with the class `as-map-wrapper`. There are no restrictions regarding the contents of a sidebar.
+Use this class to create sidebars beside the map wrapper. Sidebars are the area where your app content lives and should be placed inside an element with the class `as-app-content`. There are no restrictions regarding the contents of a sidebar.
+
+Sidebars are hidden by default in mobile devices and only will be shown when the modifier class `as-sidebar--visible` is present.
+
+
+## Class modifiers
 
 Sidebar position and size can be changed with the following class modifiers.
 
-- Position:
-  - `as-sidebar--left`: Puts the sidebar on the left of the map.
-  - `as-sidebar--right`: Puts the sidebar on the right of the map.
-- Size:
-  - `as-sidebar--l`: Makes the sidebar width to be `360px`
-  - `as-sidebar--xl`: Makes the sidebar width to be `460px`
+### .as-sidebar--left
 
+Puts the sidebar on the left of the map.
 
 
-Sidebars are hidden by default in mobile devices and only will be shown when the modifier class `as-sidebar--visible` is present.
 
 ```html
-noSource: true
-responsive: true
----
-<iframe src="/examples/layouts/sidebar/sidebar.html" style="width: 100%; height: 100%;">
+<div class="as-app"  style="height: 300px; width:100%;">
+  <main class="as-app-content">
+
+    <aside class="as-sidebar as-sidebar--left" style="background: #80B622;"></aside>
```

Comment:
Why don't we use utility classes to set all these background color instead of an inline style?